src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -164,7 +164,7 @@ const AnnotationActivity = ({
                         </div>
                         <div className="bcs-AnnotationActivity-timestamp">
                             <ActivityTimestamp date={createdAtTimestamp} />
-                            {hasVersions && !isVideoAnnotation && (
+                            {hasVersions && ((isVideoAnnotation && !isCurrentVersion) || !isVideoAnnotation) && (
```

Comment:
```suggestion
                            {hasVersions && (!isVideoAnnotation || !isCurrentVersion) && (
```

I think this can be simplified like this:

Reply:
I think that changes the logic. The component should be rendered if it either hasVersions and is a video annotation that is not for the current version or if it simply hasVersions and is not a video annotation. The component will show the page number instead of the version number for non video annotations if the annotation is for the current version of the file.

Reply:
It doesn't change the logic, just simplifies how it's written. It still renders under the same conditions:
- Has versions, but is not video annotation
- Has versions and video annotation but not current version

Reply:
Ahh. I see what you're saying. I just looked at it little more closely.

Reply:
Sure that makes sense -- if it's like that in order to make the use-case clearer then I'm okay with leaving it how it is.